public/js/src/module/gui.js

```diff
@@ -511,6 +519,9 @@ function printOcForm() {
             return new Promise( function( resolve ) {
                 setTimeout( function() {
                     $dn.trigger( 'deprintify.enketo' );
+                    textPrints.forEach( ( textPrint ) => {
+                        textPrint.dispatchEvent( events.DePrintifyText() );
```

Comment:
the event has now been renamed to DePrintify()

Reply:
The DePrintify() event should always be called (not only when !printified)

Reply:
code updated